### PR TITLE
Ren parts paper

### PR DESCRIPTION
**Pull request by david-deboer**
This just renames the parts-paper table to parts

### CODE REVIEW

**Comment by david-deboer** [comment]
I assume it is failing because I need to somehow alembic upgrade the online version?  It passes locally.

**Comment by david-deboer** [comment]
Strike that last comment -- I had errantly pushed master (unsuccessfully) rather than the needed fix to this branch...

File: alembic/versions/e33c1d5684cf_changed_parts_paper_table_to_parts.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+# revision identifiers, used by Alembic.
+revision = 'e33c1d5684cf'
+down_revision = '3d3c72ecbc0d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```
Comment (bhazelton) [inline alembic/versions/e33c1d5684cf_changed_parts_paper_table_to_parts.py]: fix this line -- it's not true

Comment (bhazelton) [reply alembic/versions/e33c1d5684cf_changed_parts_paper_table_to_parts.py]: you also need to fix the one on line 26.

**Pull request merged**